bomber.py

```diff
@@ -422,8 +422,8 @@ def selectnode(mode="sms"):
 \t Friendly Spamming
 \t Testing Your Spam Detector and more ....
 
-TBomb is not intented for malicious uses.
-"""
+TBomb is not intented for malicious uses.If for fun🤣😝😜. 
```

Comment:
The emojis are less likely to be rendered as-is in most of the terminals out there. Plus "If for fun" is not a grammatically correct phrase.